app/client/jsx/FlaskInterface.jsx

```diff
@@ -0,0 +1,82 @@
+import axios from 'axios'
+import io from 'socket.io-client'
+import { createAdventureActions } from './utils.js'
+
+const url = 'http://127.0.0.1:5000'
+const PING_INTERVAL_MS = 10000
+
+
+export class Socket {
```

Comment:
more nitpick: these are both APIs, right? One just uses WebSockets as transport, the other uses HTTP? so why not call them WebSocketApi  and HttpApi or something?

Reply:
also, I've totally forgotten, but is there a reason to not have all our API methods over WS?